pkg/discovery/endpoint/restapi/operations.go

```diff
@@ -175,11 +184,36 @@ func (o *Operation) webFingerHandler(rw http.ResponseWriter, r *http.Request) {
 		}
 
 		writeResponse(rw, resp, http.StatusOK)
+	case strings.HasPrefix(resource, fmt.Sprintf("%s%s", o.baseURL, o.transactionsPath)):
+		o.handleTransactionQuery(rw, resource)
 	default:
 		writeErrorResponse(rw, http.StatusBadRequest, fmt.Sprintf("resource %s not found", resource))
 	}
 }
 
+func (o *Operation) handleTransactionQuery(rw http.ResponseWriter, resource string) {
+	resourceSplitBySlash := strings.Split(resource, "/")
+
+	cid := resourceSplitBySlash[len(resourceSplitBySlash)-1]
+
+	resp := &WebFingerResponse{
+		Subject: resource,
+		Links: []WebFingerLink{
+			{Rel: "self", Type: "application/ld+json", Href: resource},
+			{Rel: "working-copy", Type: "application/ld+json", Href: fmt.Sprintf("%s/cas/%s", o.baseURL, cid)},
+		},
+	}
+
+	for _, v := range o.discoveryDomains {
+		resp.Links = append(resp.Links,
+			WebFingerLink{
+				Rel: "working-copy", Type: "application/ld+json", Href: fmt.Sprintf("%s/cas/%s", v, cid),
```

Comment:
Is the discovery domain the right thing to be using here? (see the "replica" at the end of this example: https://trustbloc.github.io/did-method-orb/#example-34-discover-endpoints-for-an-orb-transaction-response)